go.mod

```diff
@@ -179,6 +179,6 @@ require (
 	sigs.k8s.io/yaml v1.4.0 // indirect
 )
 
-replace github.com/vmware-tanzu/velero => github.com/openshift/velero v0.10.2-0.20250313160323-584cf1148a74
+replace github.com/vmware-tanzu/velero => github.com/openshift/velero v0.10.2-0.20250429182916-56ba9c6f9c7f
 
-replace github.com/kopia/kopia => github.com/project-velero/kopia v0.0.0-20250227051353-20bfabbfc7a0
+replace github.com/kopia/kopia => github.com/migtools/kopia v0.0.0-20250227051353-20bfabbfc7a0
```

Comment:
20bfabbfc7a0  does not belong on any branch in migtools. Do we wanna branch/tag this? or just the org change is enough.

https://github.com/migtools/kopia/commit/20bfabbfc7a0

Reply:
this is for the oadp-dev branch, so I think tracking and regularly updating other oadp-dev branches is ideal as this is development and not yet released code?